src/i_oplmusic.c

```diff
@@ -1685,7 +1685,8 @@ static const char **I_OPL_DeviceList(void)
     {
         return devices;
     }
-    array_push(devices, "OPL3 Emulation");
+    array_push(devices, "OPL3 Emulation (Vanilla)");
+    array_push(devices, "OPL3 Emulation (DMXOPL)");
```

Comment:
If a separate wad is loaded with a GENMIDI lump, should it be handled like SNDFONT and override the OPL choices? Or should it be appended? Or should it replace "Vanilla" (in which case "Default" is a more appropriate name)?

Let's use `:` to match FluidSynth naming:

```c
    array_push(devices, "OPL3 Emulation: Hello");
    array_push(devices, "OPL3 Emulation: World");
```

Reply:
> Or should it replace "Vanilla" (in which case "Default" is a more appropriate name)?

I guess that's the most sensible thing to do.

If we already call the second choice by the lump name, why not `OPL3 Emulation: GENMIDI` and `OPL3 Emulation: DMXOPL`?

Reply:
> If a separate wad is loaded with a GENMIDI lump, should it be handled like SNDFONT and override the OPL choices?

Yes, GENMIDI is just a normal lump and would be overridden. This PR adds the DMXOPL lump. We can make "fmbanks" folder, but banks I saw is in wopl format.

> If we already call the second choice by the lump name, why not OPL3 Emulation: GENMIDI and OPL3 Emulation: DMXOPL?

Done in 8081f10